server/map_rpc_server.go

```diff
@@ -57,6 +57,7 @@ type TrillianMapServerOptions struct {
 
 // TrillianMapServer implements the RPC API defined in the proto
 type TrillianMapServer struct {
+	trillian.UnimplementedTrillianMapServer
```

Comment:
Neat &ndash; I didn't know that existed!

Reply:
It's a new addition to the generated code and quite nice!